charts/alfresco-activiti-admin/Chart.yaml

```
@@ -1,11 +1,11 @@
 apiVersion: v2
 description: A Helm chart for Alfresco Activiti
 name: alfresco-activiti-admin
-version: 0.1.1
-appVersion: 24.2.0
+version: 0.2.0
```

```suggestion
version: 0.1.2
```

all the bumps are bugfixes

---

same as above